.ci/pipeline_definitions

```diff
@@ -3,6 +3,36 @@
 # SPDX-License-Identifier: Apache-2.0
 
 gardener-extension-registry-cache:
+  templates:
+    helmcharts:
+    - &registry-cache
+      name: registry-cache
+      dir: charts/gardener-extension-registry-cache
+      registry: europe-docker.pkg.dev/gardener-project/snapshots/charts/gardener/extensions
+      mappings:
+      - ref: ocm-resource:gardener-extension-registry-cache.repository
+        attribute: image.repository
+      - ref: ocm-resource:gardener-extension-registry-cache.tag
+        attribute: image.tag
+    - &registry-cache-admission-application
+      name: admission-registry-cache-application
+      dir: charts/admission/charts/application
+      registry: europe-docker.pkg.dev/gardener-project/snapshots/charts/gardener/extensions
+      mappings:
+      - ref: ocm-resource:gardener-extension-registry-cache-admission.repository
+        attribute: global.image.repository
+      - ref: ocm-resource:gardener-extension-registry-cache-admission.tag
+        attribute: global.image.tag
+    - &registry-cache-admission-runtime
+      name: admission-registry-cache-runtime
+      dir: charts/admission/charts/runtime
+      registry: europe-docker.pkg.dev/gardener-project/snapshots/charts/gardener/extensions
+      mappings:
+      - ref: ocm-resource:gardener-extension-registry-cache-admission.repository
+        attribute: global.image.repository
+      - ref: ocm-resource:gardener-extension-registry-cache-admission.tag
+        attribute: global.image.tag
```

Comment:
I assume changing from

```yaml
image: foo
```

to 

```yaml
image:
  repository: foo
  tag: bar
```

is required due to CI/CD specifics?

Because long time ago we were using the 2nd format, but then for skaffold native work we switched to the 1st one similar to how it was done for provider-local back then.

Reply:
There are separate fields in the component descriptor for `repository` and `tag` of the images. For overwriting those fields when packaging the helm chart, we have to map each field individually. Afaik there is no option for concatenation (@ccwienk please correct me if I'm wrong) .